helmfile.d/values/networkpolicies/workload/ingress-nginx-probe.yaml.gotmpl

```diff
@@ -0,0 +1,11 @@
+policies:
+  ingress-nginx:
+    {{- if .Values.wcProbeIngress.enabled }}
```

Comment:
Minor thing but not sure if this check is needed with the "installed" check earlier :thinking:

Reply:
yes, true. I will remove this